### ISSUE TITLE
How to open Kindel books? 

### ISSUE THREAD

**Issue by adjfarm** [johnfactotum/foliate#1447]
**Question:**
A clear and concise statement of your question.

**Version:**
 - Foliate version:
 - OS/Distribution and version: [e.g. Ubuntu 18.04]
 - Desktop environment: [e.g. GNOME 3.36]
 - Installation method: [e.g. Flatpak]

**Comment by adjfarm** [comment]
How do I play my Kindel books? Zorin operating Linux system.

**Comment by johnfactotum** [comment]
If you mean connecting a Kindle device, that's not something that's supported.

**Comment by adjfarm** [comment]
No, I have my books downloaded onto a hard drive. The .azw extention is listed as one used in Foliate.

"https://www.omgubuntu.co.uk/foliate-ebook-reader-for-linux" states "It can open a slew of common ebook file formats including .mobi, awz, and awz3, as well as comics and magazine .cbrand .cbzfiles."

The files on my laptop hard drive Kindel books are: .awz? .azw?
[image.png]

Is there any way to use the Foliate software?
I appreciate any help.

Sent with [Proton Mail](https://proton.me/mail/home) secure email.

**Comment by johnfactotum** [comment]
Foliate can open Mobipocket 6 and KF8 files. These files usually, though not necessarily, have the extensions .mobi, .azw, or .azw3.

The "awz" in the blog post appears to be a typo.
